erddap/content/datasets_template.xml

```diff
@@ -21,7 +21,12 @@
     <updateMaxEvents></updateMaxEvents>                               <!-- default=10 -->
     <!-- The defaults for the following tags are in messages.xml. -->
     <startHeadHtml5></startHeadHtml5>
-    <startBodyHtml5></startBodyHtml5>                                 <!-- This is often customized. -->
+    <startBodyHtml5>
+		<a href="https://bio-oracle.org/">
```

Comment:
Perhaps link to the ERDDAP itself rather than the BO website?

Reply:
I think it's OK like this